routes/routes.go

```diff
@@ -30,5 +30,7 @@ func Init() *gin.Engine {
 
 	v1.POST("process-video", controllers.ProcessVideo)
 
+	v1.POST("publish-message-kafka", controllers.PublishMessage)
```

Comment:
could you rename the function and endpoint to something related to _register video processing